src/plugin/spine.js

```diff
@@ -209,6 +209,8 @@ pc.extend(pc, function () {
                 var key = null;
                 if (texture.getSource() instanceof Image) {
                     key = texture.getSource().getAttribute("src");
+                } else if (texture instanceof pc.Texture) {
+                    key = texture.name;
```

Comment:
Texture names are not necessarily unique. Does that matter here? The comment above suggests it does.

Reply:
In that case there will be a different texture applied, but there is a workaround to rename one of those textures. Anyway, pc.Texture doesn't have any unique id.

Reply:
The plugin could track an array of referenced textures and the index in the array would be unique.